Kalk88/GrowBud
# Feature/quasar ui test

QUASAR will now be used as component library!

HOW TO TEST:

Build docker. 
Press "Sign up" in top right and create an account. Check firebase that account was created.
Press "Login" in top right and login with an existing account. Check network in dev tools and see if something is returned.
